opencti-platform/opencti-graphql/config/default.json

```diff
@@ -232,6 +232,10 @@
     "openrm_token": "",
     "openmtd_url": "",
     "openmtd_token": "",
+    "hub": {
```

Comment:
There is a consistency issue, some part of xtm hub config is under "hub" and some part is outside "xtmhub_url".
Let me think about it.